Client/Frontend/Browser/Playlist/Controllers/PlaylistCarplayController.swift

```diff
@@ -14,6 +14,12 @@ import CoreData
 
 private let log = Logger.browserLogger
 
+private enum PlaylistCarPlayTemplateID: String {
+    case Folders
+    case ItemsList
+    case Settings
```

Comment:
camel case?

Reply:
If these are caps because you need to have the raw values themselves capitalized, use `case folders = "Folders"` instead

Reply:
C++ strikes again :(
Fixed.